Content.Server/Singularity/EntitySystems/ContainmentFieldSystem.cs

```diff
@@ -16,12 +15,14 @@ public sealed class ContainmentFieldSystem : EntitySystem
 {
     [Dependency] private readonly ThrowingSystem _throwing = default!;
     [Dependency] private readonly PopupSystem _popupSystem = default!;
+    [Dependency] private readonly EventHorizonSystem _horizons = default!;
 
     public override void Initialize()
     {
         base.Initialize();
 
         SubscribeLocalEvent<ContainmentFieldComponent, StartCollideEvent>(HandleFieldCollide);
+        SubscribeLocalEvent<ContainmentFieldComponent, EventHorizonAttemptConsumeEntityEvent>(_horizons.PreventBreach);
```

Comment:
A system shouldn't create subscriptions for other systems

Reply:
Done.